src/User32.Desktop/User32.cs

```diff
@@ -13,6 +13,8 @@ namespace PInvoke
     [OfferFriendlyOverloads]
     public static partial class User32
     {
+        public const int WHEEL_DELTA = 120;
```

Comment:
What is `WHEEL_DELTA`? I don't see it come up immediately from a Bing search on MSDN. 
Can you add a comment explaining it and where you got it from?
Defining constants here is fine, so long as the native .h file is something analogous to user32.h. Otherwise, many broad Windows constants are defined in Windows.Core.